### PR TITLE
[Reference cleanup]LPS-162151

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @dantewang @tinatian,

This PR is to replace method setServletContext with field _servletContext in class extends BaseLayoutTypeControllerImpl.Thanks!
https://issues.liferay.com/browse/LPS-162151

previous PR: https://github.com/liferay-core-infra/liferay-portal/pull/1541

### CODE REVIEW

**Review by dantewang — APPROVED** [review]
@JanisZhang Code looks good, wait for QA to check // cc @zhangyan0701

**Comment by JanisZhang** [comment]
Hi @zhangyan0701 , Could you please help to scan the test results? Thank you.

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/layout/layout-type-controller/layout-type-controller-control-panel/src/main/java/com/liferay/layout/type/controller/control/panel/internal/layout/type/controller/ControlPanelLayoutTypeController.java [hunk 1]
```diff
@@ -150,7 +135,15 @@ protected void setServletContext(ServletContext servletContext) {
 
 	private static final String _VIEW_PAGE = "/layout/view/control_panel.jsp";
 
+	@Reference
```
Comment (tinatian) [inline modules/apps/layout/layout-type-controller/layout-type-controller-control-panel/src/main/java/com/liferay/layout/type/controller/control/panel/internal/layout/type/controller/ControlPanelLayoutTypeController.java:138]: @JanisZhang This is actually not related to servletContext, you should put these kind of changes to a separate commit.

**Comment by JanisZhang** [comment]
Hi @tinatian, I separate [ControlPanelLayoutTypeController.java](https://github.com/liferay-core-infra/liferay-portal/pull/1552/commits/3eac508cd042da18942b659f136eaf7c67fc726d#diff-ae1e54bdc00b4be47f80dd52a1f0aca1c10d3a1fee840b7fd280ecf35232caad) into 2 commits as you commented. Thanks.

**Comment by JanisZhang** [comment]
ci:test:sf

**Comment by JanisZhang** [comment]
ci:test:relevant

**Comment by JanisZhang** [comment]
Hi @zhangyan0701, Could you please help to scan the test result? Thank you.

**Comment by zhangyan0701** [comment]
@Kvale1733 Could you check the results? Thanks!

**Comment by dantewang** [comment]
@shuyangzhou @tinatian Forwarding this one as all tests passed and changes are direct replacements.

**Comment by dantewang** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/123002
